src/D2L.CodeStyle.Analyzers/ApiUsage/Configs/BannedConfigs.cs

```diff
@@ -12,6 +12,11 @@ internal static class BannedConfigs {
 			.Add( "GetOrg", ImmutableDictionary.Create<string, string>( StringComparer.OrdinalIgnoreCase )
 				.Add( "d2l.Settings.WebServerName", "WebServerName is being moved to Hiera data. Use IUrlFormatter or IWebServerNameProvider instead." )
 				.Add( "d2l.System.Aws.Region", "Use IOrgAwsRegionProvider instead." )
+
+				.Add( "d2l.Directory.Org.Content", "Use IFileSystemRootProvider.GetContentPath( orgId ) instead." )

```

Comment:
Can config be fetched by `Name`? I was under the impression only `ExpandedName` worked

Reply:
Yes, you are right. I pushed a new change to ban the `ExpandedName` only.